explainer.md

```diff
@@ -170,23 +183,22 @@ Challenges with the status quo include:
 
 1.  <a name="accessibility"></a>**Accessibility**: Permission UI for a
     capability is triggered through the direct use of the capability. Typically
-    JavaScript invokes permission UI, presenting an issue for both screen
+    JavaScript invokes permission UI, which can present an issue for screen
     readers and magnification users.
 
     Script attached to an existing DOM element is not interpreted by the screen
-    reader. If the DOM element was not accessibility tested and does not provide
-    sufficient explanation to its function, there is no way for a screen reader
-    user to know that the purpose of that element is to initiate access to a
-    capability. Current permissions can be accessible if properly implemented
-    and tested, PEPC is *accessible by default*.
-
-    Magnification users also struggle with the status quo. A page cannot detect
-    if a user is using OS level magnification tools (WAI for privacy reasons). A
-    user in a magnified state can easily miss the permission prompt if it falls
-    outside of their magnified viewport, and pages cannot assist these users.
-    With PEPC, the scrim and a contextually localized prompt greatly increase
-    the chance that the magnification user will observe the permission request
-    after interacting with the element.
+    reader. ARIA doesn't cover permission requests, and it requires work for
+    developers to express what an input/button will do. Current permissions can
+    be accessible if properly implemented and tested, however, PEPC has explicit
+    accessibility semantics by default.
```

Comment:
This is sorta true... but to have "explicit accessibility semantics", wouldn't it need an explicit ARIA role (which then maps to the OS's a11y model)?